x-pack/plugins/fleet/public/hooks/use_request/epm.ts

```diff
@@ -64,6 +70,16 @@ export const useGetPackages = (query: GetPackagesRequest['query'] = {}) => {
   });
 };
 
+export const useGetPackagesQuery = (query: GetPackagesRequest['query']) => {
+  return useQuery<GetPackagesResponse, RequestError>(['get-packages', query.prerelease], () =>
```

Comment:
Is there a disadvantage to just using `query` in the request key here? I know we only have one param, just thinking it could catch us out if we ever added a param, maybe not a justified concern.

Reply:
Yeah I think I was just trying to be explicit here. This can probably just be `query`.